cadl-tests/cadl/naming.cadl

```diff
@@ -0,0 +1,49 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-azure-core";
+
+using Cadl.Http;
+using Azure.Core.Operations;
+
+@summary("summary of Naming")
+@doc("description of Naming")
+@serviceTitle("Naming")
+@serviceVersion("1.0.0")
+namespace Cadl.Naming;
+
+@summary("summary of Response")
+@doc("description of Response")
+@friendlyName("DataResponse")
+model Response {
+  @summary("summary of name property")
+  @doc("description of name property")
+  name: string;
+
+  @summary("summary of data property")
+  @doc("description of data property")
+  data: bytes;
+}
+
+@summary("summary of Request")
+@doc("description of Request")
+@friendlyName("DataRequest")
```

Comment:
`@friendlyName` seems only work on Model.

In future we might need something similar to propery/parameter etc. But fine for now.